index.js

```diff
@@ -1,88 +1,119 @@
 // ***** Question 1 *****
+
+let counter = 1;
+counter = 2;
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red"); 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log(counter) ;
+// => 2
+console.log("%c----------", "color: red");
 
 
 // ***** Question 2 *****
+const name = "Gabriel"
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red");
 
-// console.log(name)
-// => "Raffy"
+console.log(name);
+//=> "Raffy"
 // name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
+//=> TypeError
+console.log("%c----------", "color: red");
 
 
 
 // ***** Question 3 *****
+function drinkWater(currentThirstLevel){
+  console.log("Man I sure am thirsty");
+  currentThirstLevel-=1; 
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel
+
+}
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+console.log(drinkWater(12))
+// => "Man I sure am thirsty"
+// => "Ahh that hits the spot"
+// => 11
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
+function sameSameButDifferent(num,maybeNum){
+  if(num === maybeNum){
+    return "same same"
+  }else if(num == maybeNum){
+    return "same same (but different)"
+  }else{
+    return "different"
+  }
+}
+
```

Comment:
This is perfect. Try refactoring the else if using parseInt and switching the == to ===

To go deeper and confirm the data type. Use typeof in your conditionals
e.g
if (typeof num === 'number' && typeof 'maybeNum' === 'number' && num === maybeNum)